src/api_server.cc

```diff
@@ -70,9 +70,15 @@ MetadataApiServer::MetadataApiServer(const Configuration& config,
                                      const MetadataStore& store,
                                      int server_threads,
                                      const std::string& host, int port)
-    : handler_(config, store),
+    : config_(config), store_(store), dispatcher_({
+        {{"GET", "/monitoredResource/"},
+         [=](const HttpServer::request& request,
```

Comment:
Is it possible to just pass HandleMonitoredResource, without doing so in a lambda? This would work in Javascript, but I know C++ is much stricter than javascript, I think anything is stricter than Javascript :-P

Reply:
No, because we also need to capture `this`. If there were no capture, you could indeed convert a function pointer into an `std::function`.